pkg/report/app_instance_tags.go

```diff
@@ -0,0 +1,91 @@
+package report
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/replicated-sdk/pkg/store"
+	tags "github.com/replicatedhq/replicated-sdk/pkg/tags"
+	tagstypes "github.com/replicatedhq/replicated-sdk/pkg/tags/types"
+	"github.com/replicatedhq/replicated-sdk/pkg/util"
+	"k8s.io/client-go/kubernetes"
+)
+
+func SendAppInstanceTags(ctx context.Context, clientset kubernetes.Interface, sdkStore store.Store, tdata tagstypes.InstanceTagData) error {
+	if err := tags.Save(ctx, clientset, sdkStore.GetNamespace(), tdata); err != nil {
+		return errors.Wrap(err, "failed to save instance tags")
+	}
+	if util.IsAirgap() {
+		return nil
+	}
+	return SendOnlineAppInstanceTags(sdkStore, tdata)
```

Comment:
I'd call `SendInstanceData` from here instead of implementing a dedicated instance-tags endpoint in the API.

Reply:
How will the vendor application create/update instance-tag without a dedicated `replicated-sdk` endpoint for the app to hit?

Reply:
Done.